chapters/interfaces.txt

```diff
@@ -2752,9 +2752,16 @@ code:PatchVertices::
 Decorating a variable with the code:PatchVertices built-in decoration will
 make that variable contain the number of vertices in the input patch being
 processed by the shader.
-A single tessellation control or tessellation evaluation shader can: read
+In a Tessellation Control Shader, this is the same as the name:patchControlPoints
+member of slink:VkPipelineTessellationStateCreateInfo.
+In a Tessellation Evaluation Shader, code:PatchVertices is equivalent to the
+tessellation control output patch size, which is controlled by the code:OpExecutionMode
+code:OutputVertices (specified in either tessellation shader).
+When the same tessellation control or tessellation evaluation shader is used
+in different pipelines, the input and/or output patch sizes may: be configured
+differently. In this case, the same tessellation control or evaluation shader can: read
 patches of differing sizes, so the value of the code:PatchVertices variable
-may: differ between patches.
+may: differ between pipelines.
```

Comment:
This seems a little bit repetitive now. Maybe it would be sufficient to say:
> The same tessellation control or tessellation evaluation shader may be used in different pipelines where the patches are of differing sizes, so the value of the code:PatchVertices variable may differ.

(Github won't let me include that as a suggestion because it spans a deleted line.)

Reply:
Agreed, let's get rid of the repetitiveness. I think this might be even more concise:

> When the same shader is used in different pipelines where the patch sizes are configured differently, the value of the code:PatchVertices variable will also differ.

How does this sound?

Reply:
Sounds good to me. Thanks.

Reply:
Done.